### PR TITLE
Initial support for building Conda package

### PR DESCRIPTION
**Pull request by yxlao**
Addresses https://github.com/IntelVCL/Open3D/issues/430

---
This change is [Reviewable](https://reviewable.io/reviews/intelvcl/open3d/514)

### CODE REVIEW

File: util/conda_package/README.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+the `pip_package`.
+
+### 1. Copy shared libraries
+
+Copy the shared libraries to the corresponding directory inside
+`Open3D/util/pip_package/open3d`. For example, on macOS with Python 3.6,
+`open3d.cpython-36m-darwin.so` shall be copied to
+`Open3D/util/pip_package/open3d/macOS`
+
+### 2. Build Conda package
```
Comment (takanokage) [inline util/conda_package/README.md]: This is good for our internal reference.
We need to add documentation for our users whom don't want to build the package from source in order to install it. 

Please take a look here: http://open3d.org/docs/tutorial/Basic/python_interface.html

My recommendation, if you all agree, is to add a section there about installing Open3D for/from Conda. That would mean also uploading a package to Conda, hopefully just one for all OSs.

Comment (yxlao) [reply util/conda_package/README.md]: Good suggestion, will add this. I'll first build and upload the package to the Anaconda repository.

**Comment by yxlao** [comment]
Anaconda packages are ready at: https://anaconda.org/open3d-admin/open3d/files

Tested on:
* Ubuntu 16.04: Python 2.7, 3.5, 3.6
* macOS: Python 2.7, 3.5, 3.6
* Windows 10 (32 & 64 bit): Python 2.7, 3.5, 3.6

Example usage:
```bash
conda create -n new_env3.6 python=3.6
source activate new_env3.6
conda install -c open3d-admin open3d
```

`open3d-admin` is a special name for us since the `open3d` channel name is already taken. This issue can be addressed later.

**Comment by ajpernalete** [comment]
Thank you!! Excellent work bro. I wish start right now to learn use open3d in my proyect. I upload something, if I can build some awsome.

**Comment by yxlao** [comment]
@ajpernalete thanks, please let us know if you run into any issues

**Comment by ajpernalete** [comment]
I fixed it. Download the TestData and paste it in C. But the documentation server is down with error 403. I test with an example.

**Comment by syncle** [comment]
I fixed the documentation webpage issue. :)

**Comment by ajpernalete** [comment]
Thank you @syncle I am doing open3d tutorial. Until now every work OK.

**Comment by qianyizh** [comment]
@takanokage @yxlao is the discussion pending addressed?

**Pull request merged**